README.md

```diff
@@ -32,6 +32,19 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+Set up instructions for Tello application :
+1)For the application, create your account in Trello and then create a board.
+We will use the app to access the board using REST APIs
+a) KEY :To begin the authentication process, you need an API key. Every Trello user is given an API key. You can retrieve your API key by logging into Trello and visiting https://trello.com/app-key/.
+b) TOKEN : You will also need a token to access your board via API. You will be directed to a page to generate user specific token once API key has been generated.
+
+2) Create an account in Postman to explore the API
+
+3) Add KEY and TOKEN to .env file instead of hardcoding in the the program
+
+4) Add the requests library to your list of poetry dependencies in
+pyproject.toml by running poetry add requests
```

Comment:
Note that this is actually only required once - when you ran this it was added to the `pyproject.toml` file in your repository, so that future developers pick up this dependency by default when they run `poetry install`